hw/ip/mbx/data/mbx.hjson

```diff
@@ -251,7 +252,8 @@
       }
       { name: "OUTBOUND_LIMIT_ADDRESS"
         desc: '''Limit Address to mark the end of the outbound mailbox memory range in the private SRAM.
-              This address is 4-byte aligned, the lower 2-bits are ignored.
+              This address is 4-byte aligned and it specifies the start address of the final valid
```

Comment:
Do you want to use the same "inclusive" wording as for the inbound limit address?

Reply:
Yes, thanks.